### DISCUSSION TITLE
tasks cannot work normally after cpu overload and then recover

### DISCUSSION THREAD

**Discussion by gochendong** [apache/dolphinscheduler#7901] [General]
I found that Dolphin has a very serious bug. I deployed it in a cluster of 2masters and 3workers. When some machines have cpu overload and then recover after a period of time, the scheduling task cannot run normally, and the dolphin service needs to be restarted.

**Comment by Tandoy** [comment]
What version of Dolphin is, Dolphin can currently perform failure recovery and thread waiting. For the configuration of master and worker memory usage, please refer to the following:
[https://dolphinscheduler.apache.org/en-us/docs/latest/user_doc/architecture/configuration.html](url)

`master.properties: master.reserved.memory`
`worker.properties: worker.reserved.memory`

**Reply by gochendong** [reply]
version is 2.0.1. The version is 2.0.1. After the CPU overload is restored, the status of all servers and scheduled tasks are displayed normally, but in fact the tasks are not scheduled

**Reply by Tandoy** [reply]
Maybe the current number of available threads is not enough to generate a task instance

`worker.properties:worker.exec.threads`

**Reply by gochendong** [reply]
All my cron tasks are only 15

**Reply by Tandoy** [reply]
Show the stack log of master and worker
